cmd/neofs-node/config.go

```diff
@@ -750,6 +751,15 @@ func initLocalStorage(c *cfg) {
 		ls.HandleNewEpoch(ev.(netmap2.NewEpoch).EpochNumber())
 	})
 
+	subscribeToContainerRemoval(c, func(e event.Event) {
+		ev := e.(containerEvent.DeleteSuccess)
+
+		err := ls.DeleteContainer(c.ctx, ev.ID)
+		if err != nil {
+			c.log.Warn("could not delete container", zap.Stringer("cID", ev.ID), zap.Error(err))
```

Comment:
message reflects the actual action too vaguely: we dont delete the container, we update local object storage after the container removal event. Otherwise, we react to `delete container` event with `delete container`

this only confirms the dubiousness of the method name

Reply:
added some details to the message